StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -96,27 +96,25 @@ public static bool CheckForAction_LanceChest_Prefix(Chest __instance, Farmer who
             }
         }
 
-        public static void ResetLocalState_PlayRailroadBoulderCutsceneIfConditionsAreMet_Postfix(GameLocation __instance)
+        public static bool EndBehaviors_AddRailroadBoulderIfIridiumBomb_Prefix(string[] split, Event __instance)
```

Comment:
please add a comment above this function describing the signature of the function that is being patched. I'm trying to do that everywhere now

Reply:
I am noticing, now that I looked into it a bit deeper, that the signature is wrong as well. What is `string[] split`? This is patching `public void skipEvent()` in `Event`, so the only parameter should be the instance

Reply:
Plus, the Patch naming convention should be `OriginaMethodName_PurposeOfThePatch_Prefix`. The method is SkipEvent(), not "Endbehaviors()"